### ISSUE TITLE
Overview page: Background svg outlines instead of fill

### ISSUE THREAD

**Issue by kajdehoop** [xrnl/aginun#201]
Background svg (tree) should use outlines instead of a filled color (probably fixed by using a border of 1px, #848484 24% opacity instead of fill on the svg) 

[Link to design](https://zeroheight.com/4eba4d3a1/p/33d529-overview/b/407259)

**Comment by aleesaan** [comment]
We would need a different svg for this, but I think the one we have now is pretty nice already: 

![image](https://user-images.githubusercontent.com/42735629/141341149-41235db8-5ec7-425b-8a6d-7d7b9afa3095.png)

Or you think the outline version is better?

**Comment by kajdehoop** [comment]
The outline is better, but I think we should be able to do that with the same svg :) With a white/none fill and a stroke of 1px, #848484 24% opacity.
https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Fills_and_Strokes

**Comment by aleesaan** [comment]
Yea I had already gave it a try but it didn't work, for instance this:  

![image](https://user-images.githubusercontent.com/42735629/141342864-e2c134f5-f3a0-4bfb-a645-c944208dad8f.png)

Looks like this:

![image](https://user-images.githubusercontent.com/42735629/141342894-24b670f1-d9cd-45f5-8fb5-170ade22bb3f.png)

Do you want to give it a try maybe and see if you can get the right svg?

**Comment by kajdehoop** [comment]
Ah alright, that looks weird indeed :P 

Can you try with this png file? This is exported from the design, I think the quality should be high enough to be sharp.
![bg tree](https://user-images.githubusercontent.com/51111316/141344248-1a9d8731-1284-4b3a-ac2b-f1149bf48213.png)

**Comment by kajdehoop** [comment]
Oh wait, I managed to export an SVG as well, I think it should work like this 🙂 

[bg tree.zip](https://github.com/xrnl/aginun/files/7522281/bg.tree.zip)

**Comment by aleesaan** [comment]
Thanks for the svg!! Just gave it a try but I don't quite like it, you can barely see it:

![image](https://user-images.githubusercontent.com/42735629/141497672-9d027074-b7e1-4245-bc5c-a6dce48c6600.png)

I think the other one is a bit more noticeable (without being too invasive) and gives more depth to the page:

![image](https://user-images.githubusercontent.com/42735629/141497806-f88951a9-9b9a-42fe-827b-0c08406269b0.png)

**Comment by kajdehoop** [comment]
I disagree, the second tree takes too much visual focus away from the cards themselves. The first one looks nice and subtle, you don't notice it right away but it makes the design look polished when you see it :)

**Comment by SamLubbers** [comment]
I also think that the background is more busy with the darker tree, but with the lighter variant I have the issue that I don't know what I'm seeing. Maybe we can just remove it all together? or find another background that improves the design without cluttering the page?

Either way, this is something that we can best learn from our users, so we can leave it for now and ask what they think :ok_hand:
